packages/web-twig/src/Resources/components/FileUploader/README.md

```diff
@@ -260,13 +267,29 @@ up the template and apply it on any attachments the user wants to upload.
 </FileUploader>
 ```
 
+With Image Preview:
+
+```twig
+<FileUploader data-spirit-toggle="fileUploader">
+  <template data-spirit-snippet="item">
+    <FileUploaderAttachment hasImagePreview />
+  </template>
+  <!-- FileUploaderInput -->
+  <!-- FileUploaderList -->
+</FileUploader>
+```
+
 ### API
 
-| Prop name    | Type     | Default  | Required | Description               |
-| ------------ | -------- | -------- | -------- | ------------------------- |
-| `fileName`   | `string` | `null`   | no       | File name                 |
-| `iconName`   | `string` | `file`   | no       | Icon shown along the file |
-| `removeText` | `string` | `Remove` | no       | Remove button text        |
+| Prop name         | Type      | Default  | Required | Description                                                 |
+| ----------------- | --------- | -------- | -------- | ----------------------------------------------------------- |
+| `editText`        | `string`  | `Edit`   | no       | Edit button text                                            |
+| `fileName`        | `string`  | `null`   | no       | File name                                                   |
+| `hasImagePreview` | `boolean` | `false`  | no       | If component has an image preview, use this in the template |
```

Comment:
I'd rather say something like the JS plugin will (attempt to) show a preview when this property is present (or `true`, whatever) in the attachment template.